lib/util.js

```diff
@@ -1136,23 +1136,11 @@ function promisify(nodeStyleFunction) {
   };
 }
 
-function promisifySave(nodeStyleFunction) {
-  return function () {
-    const args = Array.prototype.slice.call(arguments);
-    return new Promise(function (resolve) {
-      args.push(function (err, data) {
-        resolve(data);
-      });
-      nodeStyleFunction.apply(null, args);
-    });
-  };
-}
```

Comment:
This utility function had exactly one usage, where now a `.catch(() => '')` is utilized in its place.